packages/runtime/src/services/StateManager.ts

```diff
@@ -37,6 +35,8 @@ export class ExpressionError extends Error {
   }
 }
 
+export type StateManagerInterface = InstanceType<typeof StateManager>
```

Comment:
Is it the same to use `StateManager` and `StateManagerInterface`? Because they are in the same file and the interface is derived from `StateManager`.
Do we need to write a new `StateManagerInterface` and add it to `shared`?

Reply:
I think it has a little difference. When the runtime export the `StateManagerInterface` instead of `StateManager`, the outside only can use the type definition but can't generate a new instance of `StateManager`.

> Do we need to write a new StateManagerInterface and add it to shared?

I think not, because if we write the type definition alone, we need to update the interface when the `StateManager` is changed.